dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -0,0 +1,307 @@
+                        validation="required"
+                        :validation-messages="{
+                          required: 'Select a framework',
+                        }"
+                        outer-class="long"
+                        :data-test="'datapoint-framework'"
+                      />
+                    </BasicFormSection>
+                    <BasicFormSection header="Provide Contact Details">
+                      <label for="Email" class="label-with-optional">
+                        <b>Email</b><span class="optional-text">Optional</span>
+                      </label>
+                      <FormKit v-model="contact" type="text" name="contactDetails" data-test="contactEmail" />
+                      <p class="gray-text font-italic" style="text-align: left">
+                        By specifying a contact person here, your data request will be directed accordingly.<br />
+                        this increases the chances of expediting the fulfillment of your request.
+                      </p>
+                      <br />
+                      <p class="gray-text font-italic" style="text-align: left">
+                        If you don't have a specific contact person, no worries.<br />
+                        We are committed to fulfilling your request to the best of our ability.
+                      </p>
+                      <br />
+                      <label for="Message" class="label-with-optional">
+                        <b>Message</b><span class="optional-text">Optional</span>
+                      </label>
+                      <FormKit
+                        v-model="dataRequesterMessage"
+                        type="textarea"
+                        name="dataRequesterMessage"
+                        data-test="dataRequesterMessage"
+                      />
+                      <p class="gray-text font-italic" style="text-align: left">
+                        Let your contact know what exactly your are looking for.
+                      </p>
+                    </BasicFormSection>
+                  </div>
+                  <div class="col-12 flex align-items-end">
+                    <PrimeButton
+                      type="submit"
+                      label="Submit"
+                      class="p-button p-button-sm d-letters ml-auto"
+                      name="submit_request_button"
+                      @click="checkReportingPeriods()"
+                    >
+                      SUBMIT DATA REQUEST
+                    </PrimeButton>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </FormKit>
+        <div v-if="submitted" data-test="submittedDiv">
+          <template v-if="submittingSucceeded">
+            <em class="material-icons info-icon green-text">check_circle</em>
+            <h1 class="status-text" data-test="requestStatusText">Success</h1>
```

Comment:
The success message could be a bit more expressive. At least a full sentence I would say